<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotAsymmetryByLogValueDialog</class>
 <widget class="QDialog" name="PlotAsymmetryByLogValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotAsymmetryByLogValue</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>First run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstRunBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseFirstButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Last run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastRunBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseLastButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Log value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="logBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Output workspace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outWSBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Periods</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="greenBox"/>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>layoutWidget_3</zorder>
     <zorder>groupBox</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Grouping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Forward spectra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forwardBox">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Backward spectra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="backwardBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeMinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeMaxBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstRunBox</tabstop>
  <tabstop>browseFirstButton</tabstop>
  <tabstop>lastRunBox</tabstop>
  <tabstop>browseLastButton</tabstop>
  <tabstop>logBox</tabstop>
  <tabstop>outWSBox</tabstop>
  <tabstop>redBox</tabstop>
  <tabstop>forwardBox</tabstop>
  <tabstop>timeMinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
